frontend/src/metabase/containers/DashboardPicker.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import ItemPicker, { PickerValue, PickerItemId } from "./ItemPicker";
+
+export interface DashboardPickerProps {
+  // a dashboard ID or undefined
+  value?: PickerValue;
+  // callback that takes a dashboard id
+  onChange: (dashboardId: PickerItemId | undefined) => void;
+}
```

Comment:
Not sure these comments carry a lot of meaning here

Reply:
those are leftovers from the propTypes, I'll remove them :)